docs/metrics.md

```diff
@@ -90,8 +90,8 @@ Extra labels:
 * `drive` - Disk device that is being written/read.
 * `type` - Whether it's a read or write operation.
 
-#### kubevirt_vmi_storage_times_ms_total
-#### HELP kubevirt_vmi_storage_times_ms_total storage operation time.
+#### kubevirt_vmi_storage_times_ns_total
```

Comment:
Does this break the signature of the metric so dashboards can be effected?

Reply:
Yes, you're right. Forgot to mention that on release notes